src/components/Lend/LoanSearch/LoanSearchGenderFilter.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+	<fieldset class="tw-flex tw-flex-col tw-gap-2 tw-my-2">
+		<kv-radio
+			v-for="option in genderOptions"
+			:key="option.key"
+			:value="option.key"
+			v-model="selectedGender"
+			@change="setGender"
+		>
+			{{ option.title }}
+		</kv-radio>
+	</fieldset>
+</template>
+
+<script>
+import KvRadio from '~/@kiva/kv-components/vue/KvRadio';
+
+export const BOTH_KEY = 'both';
+export const BOTH_TITLE = 'All genders';
+export const FEMALE_KEY = 'female';
+export const FEMALE_TITLE = 'Women';
+export const MALE_KEY = 'male';
+export const MALE_TITLE = 'Men';
+const GENDER_KEYS = [BOTH_KEY, FEMALE_KEY, MALE_KEY];
```

Comment:
Clever use of constants, making them available for the test files too!

Reply:
I miss TypeScript in a situation like this, but using constants instead of magic strings works well enough.